frontend/src/components/RequestDialog.vue

```diff
@@ -1,17 +1,31 @@
 <template>
-  <div class="request-dialog">
-    <RequestFormInputs v-model="formResult" />
-
-    <Teleport v-if="signer" to="#action-button-portal">
-      <ActionButton
-        v-if="transferFundsButtonVisible"
-        :disabled="!formValid"
-        @click="submitRequestTransaction"
-      >
-        Transfer Funds
-      </ActionButton>
-    </Teleport>
+  <div class="request-dialog bg-transparent flex flex-col justify-between h-full">
+    <RequestSourceInputs
+      v-model="formResult"
```

Comment:
Is it a good idea to give the same model to both input components? Doesn't seem right and I would think there can be side effects. Why not having two and a computed one that merges both?